client/prisonbreak.lua

```diff
@@ -36,17 +35,16 @@ end
 --- @param y number
 --- @param z number
 --- @param text string
---- @return nil
 local function DrawText3D(x, y, z, text)
```

Comment:
Unrelated; I see this function duplicated all over. We should really add it to Shared.

Reply:
Not shared, it's a client function and also, that needs to be called every frame, so you want it to be local, otherwise it's just unnecessary extra usage because it's called from a huge table

Reply:
This is just one of those functions that has to be like that everywhere or you need to add it to ox_lib but as a global function instead of inside the lib table